code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -671,6 +671,14 @@
 /obj/machinery/cryo_cell/get_remote_view_fullscreens(mob/user)
 	user.overlay_fullscreen("remote_view", /atom/movable/screen/fullscreen/impaired, 1)
 
+/obj/machinery/cryo_cell/IsObscured()
+	. = FALSE
+	if(!isturf(loc)) //This only makes sense for things directly on turfs for now
+		return
+	for(var/atom/movable/movable in loc)
+		if(movable.flags_1 & PREVENT_CLICK_UNDER_1 && movable.density && movable.layer > layer)
+			return TRUE
```

Comment:
Copy+pasting this code from parent is kinda weak, IMO we should have something like a flag or trait like
```dm
/proc/get_turf_pixel(atom/checked_atom)
	var/turf/atom_turf = get_turf(checked_atom) 
	if(!atom_turf)
		return null
	if(checked_atom.flags_1 & IGNORE_TURF_PIXEL_OFFSET_1)
		return atom_turf
	...
```
I can imagine this would be handy in other circumstances (such as for other abnormal large sprite atoms)